packages/squeak/next.config.js

```diff
@@ -29,4 +31,4 @@ const sentryWebpackPluginOptions = {
 
 // Make sure adding Sentry options is the last code to run before exporting, to
 // ensure that your source maps include changes from all other Webpack plugins
-module.exports = withSentryConfig(moduleExports, sentryWebpackPluginOptions)
+module.exports = removeImports(withSentryConfig(moduleExports, sentryWebpackPluginOptions))
```

Comment:
Curious what the context on adding this was? (Haven't run into this package before)

Reply:
Needed to get the MD editor to work! [Context](https://github.com/uiwjs/react-md-editor#support-nextjs)